src/test/kotlin/se/uulm/snowballr/backend/repository/UserTableRepoTest.kt

```diff
@@ -377,6 +378,70 @@ class UserTableRepoTest : RepositoryTest(arrayOf(UserTable, CriterionTable, Proj
             }
     }
 
+    @Nested
+    inner class HardDeleteClearedUsers {
+        @Test
+        fun `When no cleared users exist that have reached their threshold date to be cleared, then no users are hard deleted`() =
```

Comment:
```suggestion
        fun `When no cleared users exist that have reached their threshold date to be cleared, then no users are hard-deleted`() =
```
You write soft-delete and hard delete, which is a bit inconsitent.